src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/AvoidStaticImportCheck.java

```diff
@@ -61,6 +61,13 @@
  * <pre>
  * &lt;module name="AvoidStaticImport"/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * import static java.lang.Math.*;   // violation
+ * import static java.lang.System.*; // violation
```

Comment:
Please change to static member import, now it is just a duplication of example above

Reply:
@AmrDeveloper you changed both lines, but you need to change only one. Point here is to have 2 static import lines - one with star import and second is import of static member